Cargo.toml

```diff
@@ -36,6 +36,7 @@ blst = "0.3.10"
 digest = "0.10.3"
 once_cell = "1.13.1"
 readonly = "0.2.2"
+diem-crypto-derive = "0.0.3"
```

Comment:
I'd personally avoid a diem dependency, especially because this code part is not complex to be ported (copy pasted with a comment reference) without maintainability issues. However, we can do in an upcoming PR. If we do in upcoming PR, please merge and add a github issue.

Reply:
I can do it now, but macros [need their own special crate type](https://doc.rust-lang.org/reference/procedural-macros.html), so we will have a nested crate `fastcrypto::crypto_derive.

I'll push the changes shortly.